### PR TITLE
Feature / sections

### PR DESCRIPTION
**Pull request by SedlarDavid**
- base of sections added
- needed for utb demo

### CODE REVIEW

File: src/components/section/splash/SplashTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const SplashTitle = styled.p`
```
Comment (tomesfilip) [inline src/components/section/splash/SplashTitle.tsx:3]: as it's a title we might use something semantically more accurate (one of the HTML headline tags, maybe h3?)

**Review by tomesfilip — COMMENTED** [review]
good good, just one little thingy to improve 👍

**Comment by SedlarDavid** [comment]
@tomesfilip fixed, thanks!

**Review by tomesfilip — APPROVED** [review]
💯

**Pull request merged**